src/_scss/pages/award/visualizations/_cfdaSectionTable.scss

```diff
@@ -0,0 +1,54 @@
+.cfda-section-table {
+  border: none;
+  margin-top: 0;
+  margin-bottom: rem(7);
+  font-size: rem(12);
+  .cfda-section-table__head-row {
+      border-bottom: solid 1px $color-gray-lighter;
+  }
+  .cfda-section-table__head {
+      .cfda-section-table__head-cell {
+          @import '../../recipientLanding/header';
+          @import '../../stateLanding/sorter';
+          background-color: $color-white;
+          border: 0;
+          padding: rem(15) rem(2);
+          .header-sorter {
+            .header-sorter__button {
+              svg {
+                fill: #e1e2e3;
```

Comment:
Prefer to use a variable for a pre-defined color -- this looks close to `$color-gray-warm-light`